corehq/apps/cloudcare/templates/cloudcare/config.html

```diff
@@ -20,6 +20,38 @@ <h2>{% trans 'Manage Web Apps Permissions' %}</h2>
       Please avoid using <em>both</em> this configuration and Roles & Permissions to manage access.
       Please contact <a href="mailto:{{ SUPPORT_EMAIL }}">CommCare Support</a> if you need assistance
       setting up access via Roles & Permissions.
+      <br/><br/>
+      <h5>Transitioning from the Deprecated Feature to Roles & Permissions</h5>
+
+      <p>For the simplest case where each mobile worker belongs to only one group, follow these steps:</p>
```

Comment:
Is it worth suggesting that projects with more complex use cases reach out to support?

Reply:
agreed, I wanted to avoid referring to CommCare Support twice. I shuffled it around a bit, [9fbbc69](https://github.com/dimagi/commcare-hq/pull/35245/commits/9fbbc69a9cf4344bdfa0228fc0ae280c8644fddf)

Thoughts?

Reply:
looks good to me